packages/components/.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
I'm using `npm 8.14.0`  which has peer dependency semantics which Storybook is incompatible with.
In order to work with Storybook's package structure, I'll need to run `npm` with the `--legacy-peer-deps=true` flag. We can generate an `.npmrc` which will do that automatically.

Reply:
@niloo-fs we are hardly trying to dont use legacy peer deps
is it possible to find another solution for it?

Reply:
I removed that, it is working properly without that too